plugins/utils/zellij.nix

```diff
@@ -0,0 +1,32 @@
+{
+  lib,
+  helpers,
+  config,
+  pkgs,
+  ...
+}:
+helpers.neovim-plugin.mkNeovimPlugin config {
+  name = "zellij-nvim";
+
+  originalName = "zellij.nvim";
+
+  defaultPackage = pkgs.vimPlugins.my-plugin-nvim;
+
+  maintainers = [lib.maintainers.hmajid2301];
+
+  settingsOptions = {
+    path = helpers.defaultNullOpts.mkStr "zellij" ''Zellij binary path'';
+
+    replaceVimWindowNavigationKeybinds = helpers.defaultNullOpts.mkBool false ''
+      Will set keybinds like <C-w>h to left
+    '';
+
+    vimTmuxNavigatorKeybinds = helpers.defaultNullOpts.mkBool false ''
+      Will set keybinds like <C-h> to left
+    '';
+
+    debug = helpers.defaultNullOpts.mkBool false ''
+      Will log things to /tmp/zellij.nvim
+    '';
+  };
```

Comment:
Add a `settingsExample` with the different settings set to some value (could be the default ones).

Reply:
done